content/concepts/transports/webrtc.md

```diff
@@ -1,9 +1,136 @@
 ---
 title: "WebRTC"
-description: "WebRTC is a new specification that uses QUIC to offer an alternative to WebSocket. Conceptually, it can be considered WebSocket over QUIC.Learn about WebTransport and how it is used in libp2p."
+description: "WebRTC is a protocol for real-time communication and is used to establish connections between browsers and other nodes."
 weight: 110
 aliases:
     - "/concepts/transports/webrtc"
 ---
 
-Coming soon!
+## What is WebRTC?
+
+[WebRTC (Web Real-Time Communications)](https://webrtc.org/) is a framework for real-time
+communication and in libp2p is used to establish browser-to-server and browser-to-browser
+connections between applications.
+
+WebRTC was originally designed to make audio, video, and data
+communication between browsers user-friendly and easy to implement.
+It was first developed by [Global IP Solutions (or GIPS)](http://www.gipscorp.com/).
+In 2011, GIPS was acquired by Google where the [W3C](https://www.w3.org/) started to work
+on a standard for WebRTC.
+
+It serves as a good choice for applications that need built-in support
+for media communication and do not have specific requirements for the underlying
+transport protocol.
+
+## WebRTC in libp2p
+
+In libp2p, WebRTC is used as a transport protocol to connect from browsers to other nodes.
+However, libp2p does not make use of any of WebRTC's multimedia features.
+The features employed in libp2p are:
+
+### Key features
+
+- Peer connections: WebRTC enables
+  [direct peer-to-peer connections](https://webrtc.org/getting-started/peer-connections)
+  between browsers and other nodes.
+
+- Data channels: WebRTC provides peer-to-peer [data channels](https://developer.mozilla.org/en-US/docs/Games/Techniques/WebRTC_data_channels),
+  which works on
+  [SCTP (Stream Control Transmission Protocol)](https://en.wikipedia.org/wiki/Stream_Control_Transmission_Protocol) and use
+  [SDP (Session Description Protocol)](https://en.wikipedia.org/wiki/Session_Description_Protocol) to negotiate the parameters
+  of the data channel, such as the type of data that will be sent, the codecs that will be used to encode the data, and
```

Comment:
Note that we don't use SDP to negotiate the type of data that will be sent, nor the codecs that we use. I suggest removing the mention of SDP in this paragraph.